src/ngx_http_lua_module.c

```diff
@@ -798,6 +798,7 @@ ngx_http_lua_create_main_conf(ngx_conf_t *cf)
 #if (NGX_PCRE)
     lmcf->regex_cache_max_entries = NGX_CONF_UNSET;
     lmcf->regex_match_limit = NGX_CONF_UNSET;
+    lmcf->jit_stack = pcre_jit_stack_alloc(32*1024, 32*1024);
```

Comment:
@alubbe Also, we should guard this piece with macros when PCRE is built without JIT or an older version of PCRE is used.